README.md

```diff
@@ -45,12 +45,17 @@ This library is `no_std` compatible, but uses `std` on most platforms.
 The `log` library is supported as an optional dependency. If enabled, error
 reporting will be improved on some platforms.
 
-For WebAssembly (`wasm32`), WASI and Emscripten targets are supported directly; otherwise
-one of the following features must be enabled:
+For `wasm32-unknown-unknown` target one of the following features must be
+enabled:
 
 -   [`wasm-bindgen`](https://crates.io/crates/wasm_bindgen)
 -   [`stdweb`](https://crates.io/crates/stdweb)
 
+By default compiling crate for an unsupported platform will result in a
+compilation error. You may either replace this crate with a custom one, which
```

Comment:
I think the last sentance should be replace by something like:

> If your application needs to build on targets that are not supported by `getrandom`, you can either:
>  - Use [`cfg` attributes](https://doc.rust-lang.org/rust-by-example/attribute/cfg.html) to switch to a custom implementation on unsupported targets. 
>  - Enable the `dummy` feature to have `getrandom` use an implementation that always fails on unsupported targets.
>
> We also accept pull requests to support additional targets.

This makes it clearer _why_ someone would want to do this.